translations/sk.json

```diff
@@ -128,8 +128,12 @@
   "OnboardingScreen.slides.welcome.text": "Prostredníctvom aplikácie spravovanej mestom Bratislava môžete uhradiť hodinové parkovné, či využiť bonusovú alebo návštevnícku parkovaciu kartu.",
   "OnboardingScreen.slides.welcome.title": "Vitajte v mestskej aplikácii PAAS",
   "ParkingCards.addParkingCards": "Pridať parkovacie karty",
+  "ParkingCards.cannotFindCards.cardPresent.title": "Nevidíte všetky parkovacie karty?",
+  "ParkingCards.cannotFindCards.cardPresent.text": "Viac info nájdete [tu.](https://paas.sk/casto-kladene-otazky/#ziadna-park-karta)",
+  "ParkingCards.cannotFindCards.title": "Nevidíte parkovacie karty?",
+  "ParkingCards.cannotFindCards.text": "Viac info nájdete [tu.](https://paas.sk/casto-kladene-otazky/#ziadna-park-karta)",
```

Comment:
Please use accessible link text instead of "tu"/"here".
Change `Viac info nájdete [tu.]` to for example:
`Zistite viac v [často kladených otázkach]"`
`Prejdite na [často kladené otázky]`
`Viac informácií nájdete na stránke [Často kladené otázky]`

Reply:
Changed to one of your suggestions